lebedigital/raw_data_processing/mixture/mixture_metadata_extraction.py

```diff
@@ -140,6 +142,7 @@ def extract_metadata_mixture(
             if missing_labels == ['Zusatzstoff2']:
                 logger.warning('No addition2 in raw data.')
             else:
+                logger.error('Check raw data, there are labels missing.')
```

Comment:
The error is not fully correct. If you have two errors (one resulting from a real missing label, and one from Zusatzstoff2 missing), you would only return the debug message instead of an error.

Reply:
sorry, my mistake